docs/scos/dev/guidelines/nfr-guidelines.md

```diff
@@ -0,0 +1,87 @@
+- title: Process documentation guidelines
+  link: docs/scos/dev/guidelines/process-documentation-guidelines.html
+- title: Monitorable process guidelines
+  link: docs/scos/dev/guidelines/monitorable-process-guidelines.html
+- title: Architecture performance guidelines
+  link: docs/scos/dev/guidelines/performance-guidelines/architecture-performance-guidelines.html
+---
+
+Non-Functional Requirements (NFRs) are one of the core tools for architects and developers to describe how the system under development supposed to work from technical perspective.
+Defining NFRs can be as important as defining functional requirements, which are usually done by business people (i.e. Product Owners). Missing out on them usually result in
+too late realization of certain unintended behavior that costs a lot of resources/money for the projects to deal with.
+
+## NFR guidelines
+There are several factors that can impact the success of NFR definition in a project. One key factor is the involvement of all relevant stakeholders, 
+including architects, developers, business analysts, and end users. Ensuring that all of these parties are involved in the NFR definition 
+process can help to ensure that all relevant requirements are captured and that there is a shared understanding of the technical 
+characteristics and behaviors of the system being developed.
+
+Another important factor is the use of a structured approach to NFR definition. This can involve using a specific framework or 
+methodology, such as the [ISO 25010 standard](https://iso25000.com/index.php/en/iso-25000-standards/iso-25010) or the Quality Attribute Workshop (QAW) method
+or even [PASSME](https://nick-goupinets.medium.com/passme-muster-846a9997645b). Using a structured approach can help to ensure that all relevant 
+NFRs are identified and that they are prioritized and organized in a logical manner.
+
+Finally, it is important to regularly review and update NFRs as the project progresses. As the system being developed evolves, 
+the NFRs may also need to be adjusted to reflect changes in the project's goals and objectives. Regularly revisiting and updating 
+NFRs can help to ensure that the system being developed continues to meet the technical and functional needs of the business and end users.
+
+## Template for Generic NFRs for Projects.
+
+{% info_block warningBox "Warning" %}
+
+Below you can find several generic NFRs grouped by QAs. In this list, we have provided a starting point for common NFRs that MAY be relevant
+to your project. Not all of these NFRs may be applicable in every case, so you SHOULD use this list as a guide or template to help to start
+with your own NFRs. Be sure to tailor the list to the specific requirements and constraints of your project, and to prioritize and organize
+the NFRs in a logical manner.
+
+{% endinfo_block %}
+
+### Availability
+Software architecture & design must ensure that there is no negative impact on application availability.
+
+Example NFRs:
```

Comment:
No, these are NOT the NFRs. The NFR is the opening sentence: "Software architect ... on application availaiblity." These are examples on how to fulfil the NFR.